rdkit/Chem/FeatMaps/FeatMapParser.py

```diff
@@ -162,30 +160,19 @@ def ParseFeatPointBlock(self):
       p = FeatMapPoint.FeatMapPoint()
 
       i = 0
```

Comment:
This is no longer required

Reply:
Thanks you for your review. I will fix it now. I have remembered to edit it?